src/follower.py

```diff
@@ -137,8 +132,8 @@ def reset_camera(self):
 
         The default position is tilted to 120 degrees and panned to 90 degrees.
         """
-        self._camera.turn_down(120)
-        self._camera.pan_servo.write(90)
+        self._camera.tilt(0)
```

Comment:
Change the comment above this too. We're not tilting to 120 degrees here, are we?